modules/metering-store-data-in-s3.adoc

```diff
@@ -5,6 +5,8 @@
 [id="metering-store-data-in-s3_{context}"]
 = Storing data in Amazon S3
 
+Metering can use an existing Amazon S3 bucket or create a new bucket for storage. Metering does not manage or delete any S3 bucket data. When uninstalling metering, any S3 buckets used to store metering data must be manually cleaned up.
```

Comment:
Can you uninstall metering? There's no topic for it, and this information would be far more helpful in an uninstalling topic.

~new~

Consider refocusing the last sentence on the fact that you must manage the S3 data yourself and using uninstalling as an example.

Reply:
Great suggestion. We were discussing the same thing on the metering team. I'm going to do a separate PR for a section on "Deleting Reports". I'll add in a section for "Uninstalling Metering."